### DISCUSSION TITLE
TreeEntity封装了菜单数据后返回给前端的菜单数据没有按照sortValue排序了

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#273] [02.提问]
有个bug。TreeEntity封装了菜单数据后返回给前端的菜单menu chidren里数据没有按照sortValue排序了

TreeUtil  .buildTre  方法里并没有按sortValue 升序去统一组装数据

**Comment by zuihou** [comment]
TreeUtil .buildTre（list）


传入的list要先按 sortValue排序

**Reply by zqb666** [reply]
<img width="775" alt="image" src="https://user-images.githubusercontent.com/6092525/231715686-be7e6aff-704c-4487-8c5d-a2545e1c6c10.png">
传入的 treeList还是排好序的

**Reply by zqb666** [reply]
buildTree 后 chidren里面的数据就不按照顺序排了
